src/errors/RefNotFoundError.ts

```diff
@@ -0,0 +1,8 @@
+import { PrismicError } from "./PrismicError";
+
+type RefNotFoundErrorAPIResponse = {
+	type: "api_notfound_error";
+	message: string;
+};
+
+export class RefNotFoundError extends PrismicError<RefNotFoundErrorAPIResponse> {}
```

Comment:
Can we make this extend `NotFoundError` perhaps? This makes it a non-breaking change and is still semantically correct(?)

Reply:
The API was returning 403 responses before the recent change, which means a `ForbiddenError` was thrown instead. We could still extend from `ForbiddenError` instead to prevent the breaking change.

What do you think? I'm okay with extending `ForbiddenError` provide we add a TODO to change it to `PrismicError` in v8. In my opinion, `NotFoundError` should be completely separate from `RefNotFoundError` since the API's payloads may be different.

Reply:
I updated the PR ([see this commit](https://github.com/prismicio/prismic-client/pull/327/commits/cdbedd79bfa31e0e5f5aff249f1e5ca0403af57e)) and updated the PR body. I can still rollback or change the implementation if you have any other thoughts. 🙂

Reply:
OK yes, perfect!